simple_ldap_user/simple_ldap_user.admin.inc

```diff
@@ -298,13 +298,12 @@ function simple_ldap_user_import($form, &$form_state) {
   
   $new_names = $users;
   $sql = "select name from {users} where name= :name";
-  foreach ($new_names as $item){
-     $result = db_query($sql, array(':name'=>$item));
-     if ($result) {
-        while ($row=$result->fetchAssoc()) {
-            // Filter users fetched from ldap database and mysql database
-
-            unset ($new_names[strtoupper($row['name'])]);
+  foreach ($new_names as $item) {
+    $result = db_query($sql, array(':name'=>$item));
+    if ($result) {
+      while ($row=$result->fetchAssoc()) {
+      // Filter users fetched from ldap database and mysql database
```

Comment:
If you want this comment to describe the entire `while` control structure, keep it at this indenting but move it above.  If you want the comment to describe the interior of the `while`, indent it to make it a child of this block.